Resources/Prototypes/Entities/Objects/Materials/Sheets/glass.yml

```diff
@@ -9,6 +9,9 @@
     sprite: Objects/Materials/Sheets/glass.rsi
   - type: Item
     sprite: Objects/Materials/Sheets/glass.rsi
+  - type: StaticPrice
+    price: 0
+  - type: MaterialPrice
```

Comment:
im a little confused why it has `StaticPrice` 0 but also `MaterialPrice`, seems like these would be mutually exclusive, or that the static price shouldnt have to be specified at all

Reply:
All items have a default static price, so i'm overriding it.